tests/scripts/density.js

```diff
@@ -172,45 +81,41 @@ function applySpinApps(kubernetes, spinApps) {
  * @returns {string} The URL to be used for the test.
  */
 export function setup() {
-  let endpoints = [];
-  // TODO: get the number of batches from the Options
-  for (let i = 1; i <= totalApps; i++) {
-    let appName = `${testName}-${i}`;
-    let svc = deploy.serviceEndpointForApp(appName, namespace, route);
-    endpoints.push(new TestEndpoint(appName, svc));
-  }
-  return endpoints;
-}
-
-export function deployApps() {
   const kubernetes = new Kubernetes();
-  let batchSize = `${__ENV.BATCH_SIZE}` != "undefined" ? `${__ENV.BATCH_SIZE}` : 10;
-  let batchNumber = `${__ENV.BATCH_NUMBER}` != "undefined" ? `${__ENV.BATCH_NUMBER}` : 0;
-  console.log("Deploying apps for batch " + batchNumber);
+  let allApps = deploy.getSpinApps(kubernetes, namespace).length;
+  let startQuantity = allApps == 0 ? 0 : allApps;
```

Comment:
Out of curiosity: can this just be `let startQuantity = allApps` then?  Or not even need `startQuantity` in that case?